inactive_session_timeout/models/ir_config_parameter.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 ACSONE - Cédric Pigeon
+# Copyright 2017 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models, tools
+
+DELAY_KEY = 'inactive_session_time_out_delay'
+IGNORED_PATH_KEY = 'inactive_session_time_out_ignored_url'
+
+
+class IrConfigParameter(models.Model):
+    _inherit = 'ir.config_parameter'
+
+    @api.model
+    @tools.ormcache(skiparg=0)
```

Comment:
Don't use skiparg, [it's kept for backwards compatibility, but the new and preferred way of caching is by argument names](https://github.com/odoo/odoo/blob/10.0/odoo/tools/cache.py#L34-L47).

In any case, IMHO this whole file is not needed. [You should just use Odoo's native `get_param` method, which already has caching enabled](https://github.com/odoo/odoo/blob/10.0/odoo/addons/base/ir/ir_config_parameter.py#L53-L68).